tests/sentry/api/endpoints/test_organization_dashboards.py

```diff
@@ -861,3 +861,76 @@ def test_post_dashboard_with_widget_filter_requiring_environment(self):
         }
         response = self.do_request("post", f"{self.url}?environment=mock_env", data=data)
         assert response.status_code == 201, response.data
+
+    def test_post_dashboard_with_widget_split_datasets(self):
```

Comment:
should we write a test for `put` as well?

Reply:
The list endpoint only supports `GET` and `POST`! The `PUT` test is on the detail endpoint where you're working with a specific dashboard instance (in `tests/sentry/api/endpoints/test_organization_dashboard_details.py`), it's basically the same test